src/action/macos/create_determinate_nix_volume.rs

```diff
@@ -187,8 +187,8 @@ impl Action for CreateDeterminateNixVolume {
 
         let mut command = Command::new("/usr/local/bin/determinate-nixd");
         command.args(["--stop-after", "mount"]);
-        command.stderr(std::process::Stdio::piped());
-        command.stdout(std::process::Stdio::piped());
+        command.stderr(std::process::Stdio::inherit());
+        command.stdout(std::process::Stdio::inherit());
```

Comment:
?

This means the `command.output()` below will now return nothing, because it will inherit stderr and stdout from the installer process (meaning it'll be writing to a terminal instead of being captured by Rust)

Reply:
```suggestion
        command.stderr(std::process::Stdio::piped());
        command.stdout(std::process::Stdio::piped());
```